dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -117,7 +117,7 @@
     private static final String BITBUCKET_URL = "https://bitbucket.org/";
     private static final String GITLAB_URL = "https://gitlab.com/";
     private static final String ORCID_URL = "https://orcid.org/";
-    private static final TOSVersion CURRENT_TOS_VERSION = TOSVersion.TOS_VERSION_1;
+    private static final TOSVersion CURRENT_TOS_VERSION = TOSVersion.TOS_VERSION_2;
```

Comment:
I believe this will cause issues if not merged alongside https://github.com/dockstore/dockstore-ui2/commits/hotfix/2.7.4, specifically the commit https://github.com/dockstore/dockstore-ui2/commit/77d80dca3fcde43ac43872a4c07845648038c3bb. 

This will cause a 'your session has expired' loop when logging in.

Reply:
Yes UI PR https://github.com/dockstore/dockstore-ui2/pull/1173 is in progress